src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Utf8JsonWriterTests.cs

```diff
@@ -4067,9 +4068,9 @@ public void WriteInvalidDepthPartial(bool formatted, bool skipValidation)
         [InlineData(true, false, ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")]
         [InlineData(false, true, ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")]
         [InlineData(false, false, ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>")]
-        public void WriteComments(bool formatted, bool skipValidation, string comment)
+        public void WriteCommentsInArray(bool formatted, bool skipValidation, string comment)
         {
-            string expectedStr = GetCommentExpectedString(prettyPrint: formatted, comment);
+            string expectedStr = GetCommentInArrayExpectedString(prettyPrint: formatted, comment);
```

Comment:
Per my comment above, would it be possible to include a couple of smoke tests that check indentation against a couple of string literals rather than constructing the expected string using test logic? Both might contain bugs or could regress.

Reply:
Agreed, I'm looking at the test and it's really hard to figure out at first glance what this test is actually testing. I'd prefer couple of hardcoded string cases as well

Reply:
Even I agree :) Changed to use string literal now.
Just note in advance when you view inside string literal result: 

> /*Comment in the middle of object*/,
> /*Comment in the middle of array*/,

This behavior is existing one, I did not change and just include them into test cases.
Both #75075 and #52408 should have be covered by test cases.